src/ports/postgres/modules/kmeans/test/kmeans.sql_in

```diff
@@ -91,6 +91,59 @@ SELECT * FROM kmeans('kmeans_2d', 'array[x,y]', 'centroids', 'array[x,y]');
 SELECT * FROM kmeanspp('kmeans_2d', 'array[x,y]', 10);
 SELECT * FROM kmeans_random('kmeans_2d', 'arRAy [ x,y]', 10);
 
+-- Test kmeanspp_auto
```

Comment:
we should add asserts to check for the values in the two output tables for at least some of these select statements.

Reply:
Also I found out that if selection_algo is passed as both, then in the output summary table we store both as the selection_algo instead of elbow.

Reply:
We can add asserts like
1. Count of the output table 
1. Make sure all the k values are as expected
1. Other metadata like `selection_algo`, `num_iterations` etc.
etc.